python/wait/requirements.txt

```diff
@@ -0,0 +1 @@
+git+https://github.com/arcalot/arcaflow-plugin-sdk-python.git@0e878bbe60b2a4e8e22532064e88c750e990ef8b
```

Comment:
is there a reason to point to a specific commit?

Reply:
in absence of a tag, just taking a version which has been tested with the plugin. Will be replaced with standard latest pip release as a new release with included fixes for float validation will be available.

Reply:
ok